src/util/formChangeCount.tsx

```diff
@@ -0,0 +1,143 @@
+import {
+  ConfigurationRowFormikProps,
+  ConfigurationRowFormikValues,
+} from "components/ConfigurationRow";
+import { FormDevice, FormDeviceValues } from "util/formDevices";
+import { ResourceLimitsFormValues } from "components/forms/ResourceLimitsForm";
+import { InstanceEditDetailsFormValues } from "pages/instances/EditInstance";
+
+const getPrimitiveFieldChanges = (
+  formik: ConfigurationRowFormikProps,
+): number => {
+  let changeCount = 0;
+
+  const ignoredFields = [
```

Comment:
small improvement, this could be a `Set` which is better for lookup and `Set.has()` is not any more complex than `Array.includes()` wdyt?